WordPress/Sniffs/WP/EnqueuedResourcesSniff.php

```diff
@@ -22,6 +25,10 @@
  * @since   0.3.0
  * @since   0.12.0 This class now extends the WordPressCS native `Sniff` class.
  * @since   0.13.0 Class name changed: this class is now namespaced.
+ * @since   3.0.0  Extended tokens array to include inline HTML and
```

Comment:
The target token array has actually not been extended, just changed.

The difference is that in the old logic, the sniff would look at the tokens containing the content of heredoc/nowdoc structures - `T_HEREDOC`, `T_NOWDOC` -, while now it looks for the start tokens of those.
Other than that, the same tokens are being sniffed (`T_CONSTANT_ENCAPSED_STRING`, `T_DOUBLE_QUOTED_STRING` and `T_INLINE_HTML`.

As a side-effect, this makes the sniff more efficient as the heredoc/nowdoc start tokens exist a lot less frequently than the content tokens for the same.

Either, that means the changelog entry for that can be removed. It's just an implementation detail.

```suggestion
 * @since   3.0.0  Added a check for a complete text string in the case where it
 *                 spans two lines.
```

As for what's left in the class changelog entry - I don't really think this entry is needed.
The sniff does nothing new, it just does what it was supposed to do _better_.